scipy/stats/tests/test_distributions.py

```diff
@@ -2969,6 +2969,56 @@ def test_entropy_transposed(self):
                                   stats.entropy(pk, qk, axis=1))
 
 
+class TestDifferentialEntropy(object):
+    """
+    Results are compared with the R package vsgoftest.
+        
+    """
+
+    def test_differential_entropy_base(self):
+        
+        random_state = np.random.RandomState(0)
+        values = random_state.standard_normal(100)
+        entropy = stats.differential_entropy(values)
+        assert_(abs(entropy - 1.122044) < 1.e-5)
```

Comment:
we have a slight preference for plain `assert` now that we're using `pytest`

Reply:
Changed to `assert`.